code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -305,10 +305,10 @@
 	// Milk is good for humans, but bad for plants. The sugars cannot be used by plants, and the milk fat harms growth. Not shrooms though. I can't deal with this now...
 /datum/reagent/consumable/milk/on_hydroponics_apply(obj/item/seeds/myseed, datum/reagents/chems, obj/machinery/hydroponics/mytray, mob/user)
 	. = ..()
-	if(chems.has_reagent(type, 1))
-		mytray.adjust_waterlevel(round(chems.get_reagent_amount(type) * 0.3))
+	if(chems.has_reagent(src.type, 1))
```

Comment:
Revert all these, they are not necessary and go against our style guide

Reply:
The `src.`

Reply:
so all of them since when debugging it made no difference only between src